web/src/views/FashionAnnotationInfoView.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export default function FashionAnnotationInfoView() {
+    const onFinish = values => {
```

Comment:
I don't think this component needs an onFinish hook.

Reply:
I agree i will remove it